packages/cli/tests/e2e/bot/CommandBotHappyPathCommon.ts

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @author Xiaofu Huang <xiaofhua@microsoft.com>
+ */
+
+import fs from "fs-extra";
+import path from "path";
+
+import { AadValidator, BotValidator } from "../../commonlib";
+
+import {
+  execAsync,
+  execAsyncWithRetry,
+  getSubscriptionId,
+  getTestFolder,
+  getUniqueAppName,
+  cleanUp,
+  readContextMultiEnv,
+} from "../commonUtils";
+import AppStudioLogin from "../../../src/commonlib/appStudioLogin";
+import { environmentManager } from "@microsoft/teamsfx-core";
+
+import { it } from "../../commonlib/it";
+
+export function happyPathTest(runtime: "dotnet" | "nodejs"): void {
+  describe("Provision", function () {
+    const testFolder = getTestFolder();
+    const appName = getUniqueAppName();
+    const subscription = getSubscriptionId();
+    const projectPath = path.resolve(testFolder, appName);
+    const envName = environmentManager.getDefaultEnvName();
+
+    const env = Object.assign({}, process.env);
+    env["TEAMSFX_CONFIG_UNIFY"] = "true";
+    env["BOT_NOTIFICATION_ENABLED"] = "true";
```

Comment:
Nit: Just for syncing information for myself - Feel free to resolve this.
I remember some of these env vars are turned on by default. It seems these env vars are still working here, why?